fabric/gradle.properties

```diff
@@ -3,11 +3,11 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://fabricmc.net/versions.html
-	minecraft_version=1.18.1
-	yarn_mappings=1.18.1+build.22
-	loader_version=0.12.12
+	minecraft_version=1.18.2
+	yarn_mappings=1.18.2+build.2
+	loader_version=0.13.3
```

Comment:
The original setup works well in 1.18.1, 1.18.2, even in 1.19 snapshots. Rise the minimum loader version could break the compatibility.